ports/superdungeonmuncher/README.md

```diff
@@ -0,0 +1,25 @@
+# Super Dungeon Muncher
+This is a port of Super Dungeon Muncher by nexy via PortMaster. It uses https://github.com/JohnnyonFlame/gmloader-next.
+
+# Installation
+Download the game from https://n-nexy.itch.io/super-dungeon-muncher and extract it. Put the data.win from the extracted folder into your ports/superdungeonmuncher folder. Also works with STEAM version. Browse local files and find data.win, put into your ports/superdungeonmuncher folder.
+
+## Default Gameplay Controls
+| Button | Action |
+|--|--|
+|D-PAD|Move|
+|LEFT/RIGHT ANALOG|not working|
```

Comment:
If you use a gptk file with gptokeyb, you can map the analog stick to keyboard keys wasd and have movement. If BXY do the same thing A does, just show ABXY in one row.

Reply:
thanks! didn't know that